src/middlewared/middlewared/plugins/interface/lagg.py

```diff
@@ -0,0 +1,63 @@
+from middlewared.service import Service
```

Comment:
Prety minor but can we rename it to `lag` or linkaggregation?

FreeBSD is the only operating system I found to call that kind of interface `lagg`

Reply:
It's named lagg in database and API, is it really worth it to rename?

Reply:
Where do we have lagg in the API? I made sure we used LINK_AGGREGATION instead

Reply:
Ok, I am wrong, but it's still lagg in the database, rename that too?

Reply:
I dont think we have to go as far as renaming the database